runtime/src/lib.rs

```diff
@@ -400,9 +400,9 @@ impl parachain_staking::Config for Runtime {
 	type Currency = Balances;
 	type BlocksPerRound = BlocksPerRound;
 	type BondDuration = BondDuration;
-	type TotalSelectedCandidates = TotalSelectedCandidates;
+	type MinSelectedCandidates = MinSelectedCandidates;
 	type MaxNominatorsPerCollator = MaxNominatorsPerCollator;
-	type MaxCollatorsPerNominator = TotalSelectedCandidates;
+	type MaxCollatorsPerNominator = MinSelectedCandidates;
```

Comment:
@artkaseman what number should we increase this to? It is set to 8 now and bounds the number of nominations per nominator.